ui/apps/platform/src/services/PoliciesService.ts

```diff
@@ -206,6 +207,14 @@ export function exportPolicies(policyIds: string[]): Promise<void> {
     });
 }
 
+export function savePoliciesAsCustomResource(policyIds: string[]): Promise<void> {
```

Comment:
Remind me about pro and con of `Promise<void>` and `Promise<Empty>` as return type?

Reply:
In this case, the `saveFile` function never explicitly resolves its promise with a value, so it essentially is the same as `Promise<undefined>`, but I like `Promise<void>` better as it communicates a fire-and-forget type action, while `Promise<undefined>` implies that we are explicitly returning nothing.

I typically use `Promise<Empty>` for APIs that return an empty object `{}`, that we then resolve in the service function.

In all three cases, the return value is not intended to be used but I think it is worthwhile to separate them based on the small semantic differences.

Reply:
Thank you! Yes, I agree that it is worthwhile.